### DISCUSSION TITLE
Mount problems

### DISCUSSION THREAD

**Discussion by wkirby** [abiosoft/colima#1147] [Q&A]
Hi there,

I'm trying to muddle my way through what I believe to be filesystem mounting problems. We work with a number of clients, so we have a lot of different dev environments which all get managed with docker compose. These projects all live in subdirectories of `~/Dev`. With docker desktop, I can add `~/Dev` to the mounts settings, and all these projects work.

With colima, I have two different problems: some projects won't start because they can't find files they want to mount, and other projects will start, but don't have any files on the container file system. All projects are brought up with `docker compose` and all work as expected on docker desktop with only the one setting change identified above.

For colima, I'm starting with:

```
colima start --cpu 4 --memory 16 --vm-type vz --mount-type virtiofs --mount ~/Dev:w
```

Any insight into what I might be doing wrong? Thanks!

**Comment by abiosoft** [comment]
Specifying `--mount ~/Dev:w` is not needed, Colima mounts `$HOME` by default.

Can you share more details? e.g. the absolute path of `~/Dev` and the docker compose file.

**Reply by wkirby** [reply]
Thanks for the response, I didn't think it should be needed, but... trying anything.

Full paths:

```
/Users/wyatt/Dev

/Users/wyatt/Dev/client/project/Dockerfile
/Users/wyatt/Dev/client/project/docker-compose.yml
```

**Reply by abiosoft** [reply]
It should just work.

Do you mind resetting the Colima instance and starting without specifying the `mount` flag?
